mne/tests/test_docstring_parameters.py

```diff
@@ -62,6 +62,9 @@ def get_name(func):
     'mne.io.Info',  # Parameters
     'mne.io.write',  # always ignore these
     # Deprecations
+    'mne.utils.dics_source_power',
+    'mne.utils.dics',
+    'mne.utils.dics_epochs',
```

Comment:
FYI you might not need to add these if you put the `@verbose` decorator below the `@deprecated` decorator, but it doesn't make much practical difference

Reply:
that would be neat. I'll give it a try.